frontend/src/component/project/Project/ProjectStats/Feedback.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, VFC } from 'react';
+import { Box, Paper, Button } from '@mui/material';
+import { usePlausibleTracker } from 'hooks/usePlausibleTracker';
+
+interface IFeedbackProps {
+    id: string;
+}
+
+export const Feedback: VFC<IFeedbackProps> = ({ id }) => {
+    const [selected, setSelected] = useState<'yes' | 'no'>();
+    const { trackEvent } = usePlausibleTracker();
+
+    const onTrackFeedback = (value: 'yes' | 'no') => {
+        setSelected(value);
+        trackEvent('project_overview', {
+            props: {
+                eventType: id,
+                wasHelpful: value === 'yes',
+            },
+        });
+    };
```

Comment:
Should we record this in the DB/localstorage? Or should we accept that a user can provide the feedback multiple times?

The drawback of localstorage is that we clear it on logout.